src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -377,8 +379,8 @@ export class SavedObjectsService
      * HTTP server running without an Elasticsearch server being available.
      * So, when the `migrations.skip` is true, we skip migrations altogether.
      */
-    const cliArgs = this.coreContext.env.cliArgs;
-    const skipMigrations = cliArgs.optimize || this.config.migration.skip;
+    const skipMigrations =
```

Comment:
could you add a comment why we shouldn't run migration. It's not really obvious... probably we should run dev cluster master with `--skip-migration=true` instead?

Reply:
Actually we might be able to drop this flag...I think the master process also specifies plugin.initialized=false